integration/spark/src/main/scala/org/apache/spark/sql/execution/strategy/DDLHelper.scala

```diff
@@ -398,9 +398,7 @@ object DDLHelper {
   }
 
   def showTables(showTablesCommand: ShowTablesCommand): Seq[SparkPlan] = {
```

Comment:
is this method really required? can't we just directly call the command class from `DDLStrategy `class? Why to add an intermediate method?

Reply:
Not required. Have removed it